app/Models/Category.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Category extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    protected $fillable = ['name'];
+
+    public function items()
+    {
+        return $this->hasMany(Item::class, 'category_id');
```

Comment:
em chỉ cần ghi như dưới là đươc, trừ trường hợp trong table items trường khóa ngoại khai báo 1 tên khác biệt thì mới cần khai báo như bên trên của em.
$this->hasMany(Item::class);